MatrixSDKTests/MXRestClientTests.m

```diff
@@ -377,11 +378,11 @@ - (void)testRoomAddAlias
             // Test with a valid alias
             [bobRestClient2 addRoomAlias:roomId alias:correctAlias success:^{
                 
-                [bobRestClient2 roomIDForRoomAlias:correctAlias success:^(NSString *roomId2) {
+                [bobRestClient2 resolveRoomAlias:correctAlias success:^(MXRoomAliasResolution *resolution) {
                     
-                    XCTAssertNotNil(roomId2);
-                    XCTAssertNotEqual(roomId2.length, 0);
-                    XCTAssertEqualObjects(roomId2, roomId, @"Mapping from room alias to room ID is wrong");
+                    XCTAssertNotNil(resolution);
+                    XCTAssertNotEqual(resolution.roomId.length, 0);
+                    XCTAssertEqualObjects(resolution.roomId, roomId, @"Mapping from room alias to room ID is wrong");
                     
```

Comment:
I wonder if we could test something relevant on the servers list 🤔

Reply:
Oh yea! I was going to do that and then it slipped my mind, thanks!